### ISSUE TITLE
Add checkable ToolStrip menu items and buttons to test applications

### ISSUE THREAD

**Issue by dmitrii-drobotov** [dotnet/winforms#9005]
### .NET version

.NET 8

### Did it work in .NET Framework?

No

### Issue description

Following up on recent accessibility improvements for checkable ToolStripMenuItem and ToolStripButton (#8307, #8961), we should add samples of such controls into our test apps. 

#### WinFormsControlsTest app

* ToolStripMenuItem. There is already a checked item on "Menus" page:
![image](https://user-images.githubusercontent.com/102954094/233125000-a851d44c-b173-41c4-a03b-22a130231708.png)
but more cases should be included:
    * `ToolStripMenuItem` with `CheckState = CheckState.Unchecked` and `CheckOnClick = true`
    * `ToolStripMenuItem` with `CheckState = CheckState.Indeterminate` and `CheckOnClick = true`
    * `ToolStripMenuItem` with `CheckState = CheckState.Indeterminate` and `CheckOnClick = false`
    
* ToolStripButton. Different cases are already covered on "ToolStrips" page:
![image](https://user-images.githubusercontent.com/102954094/233126174-3b1dbaf6-85c2-4a81-95e5-9d44f92473bb.png)

#### Accessibility_Core_App

* ToolStripMenuItem. There are no samples of checked items on MenuToolbars_Controls - StripControls page, therefore we should add following items:
    * `ToolStripMenuItem` with `CheckState = CheckState.Unchecked` and `CheckOnClick = true`
    * `ToolStripMenuItem` with `CheckState = CheckState.Checked` and `CheckOnClick = true`
    * `ToolStripMenuItem` with `CheckState = CheckState.Checked` and `CheckOnClick = false`
    * `ToolStripMenuItem` with `CheckState = CheckState.Indeterminate` and `CheckOnClick = true`
    * `ToolStripMenuItem` with `CheckState = CheckState.Indeterminate` and `CheckOnClick = false`
    
* ToolStripButton. There are also no checked buttons on MenuToolbars_Controls - StripControls page, therefore we should add a set of similar cases:
    * `ToolStripButton` with `CheckState = CheckState.Unchecked` and `CheckOnClick = false`
    * `ToolStripButton` with `CheckState = CheckState.Unchecked` and `CheckOnClick = true`
    * `ToolStripButton` with `CheckState = CheckState.Checked` and `CheckOnClick = true`
    * `ToolStripButton` with `CheckState = CheckState.Checked` and `CheckOnClick = false`
    * `ToolStripButton` with `CheckState = CheckState.Indeterminate` and `CheckOnClick = true`
    * `ToolStripButton` with `CheckState = CheckState.Indeterminate` and `CheckOnClick = false`

### Steps to reproduce

N/A

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang - would your team be able to extend the accessibility test app with the suggested cases?

**Comment by Amy-Li03** [comment]
@Tanya-Solyanik have added suggested cases about ToolStripMenuItem and ToolStripButton for accessibility .NET 8.0 test app.
![image](https://user-images.githubusercontent.com/62929087/233315683-76ed9b4a-59a9-489a-8c60-3fadb43f9ca8.png)

**Comment by dmitrii-drobotov** [comment]
@Amy-Li03 looks good! Can you also create a pull request with these changes to the [form](https://github.com/dotnet/winforms/blob/024a9ab98d4bea535ce9fe6c473326bcd00464b7/src/System.Windows.Forms/tests/AccessibilityTests/Menu_Toolbars_controls.Designer.cs) in this repo?

**Comment by dmitrii-drobotov** [comment]
Both test apps were updated.

**Comment by John-Qiao** [comment]
Verified this on .NET 8.0 latest build: 8.0.100-preview.5.23226.2, issue is fixed that the checkable ToolStrip menu items and buttons added in test application as below:
 
<img width="555" alt="9005-Testresult" src="https://user-images.githubusercontent.com/45864985/234769109-76d771ea-2ebb-4b64-9529-70080e2fdfe3.png">

**Comment by Amy-Li03** [comment]
On WinForms repo from release/8.0-Preview4 branch, AccessibilityApp contains checkable ToolStrip menu items and ToolStripButtons, the result is same as above.
